node/grpc/server.go

```diff
@@ -154,6 +157,26 @@ func (s *Server) NodeInfo(ctx context.Context, in *pb.NodeInfoRequest) (*pb.Node
 	return &pb.NodeInfoReply{Semver: node.SemVer, Os: runtime.GOOS, Arch: runtime.GOARCH, NumCpu: uint32(runtime.GOMAXPROCS(0)), MemBytes: memBytes}, nil
 }
 
+func (s *Server) GetChunks(context.Context, *pb.GetChunksRequest) (*pb.GetChunksReply, error) {
+	return nil, status.Errorf(codes.Unimplemented, "method GetChunks not implemented")
+}
+
+func (s *Server) GetChunk(context.Context, *commonpb.ChunkKey) (*commonpb.ChunkData, error) {
+	return nil, status.Errorf(codes.Unimplemented, "method GetChunk not implemented")
+}
+
+func (s *Server) GetHeader(context.Context, *commonpb.BlobKey) (*pb.GetBlobHeaderReply, error) {
+	return nil, status.Errorf(codes.Unimplemented, "method GetHeader not implemented")
+}
+
+func (s *Server) StreamHeaders(pb.Retrieval_StreamHeadersServer) error {
```

Comment:
I assume this means there could be support for long standing GRPC connections ?

Reply:
That was the idea. I've never set up a system with long lived GRPC connections, so I don't know if there are any pitfalls we need to worry about. Is this something you think we should reconsider from a design perspective?